pkg/sql/execinfrapb/processors_changefeeds.proto

```diff
@@ -81,6 +81,12 @@ message ChangeFrontierSpec {
   // changefeed at that timestamp.
   repeated roachpb.Span tracked_spans = 1 [(gogoproto.nullable) = false];
 
+  // Change frontier legacy checkpoint
+  optional cockroach.sql.jobs.jobspb.ChangefeedProgress.Checkpoint checkpoint = 6;
+
+  // Change frontier span level checkpoint
+  optional cockroach.sql.jobs.jobspb.TimestampSpansMap span_level_checkpoint = 7;
```

Comment:
nit: add this to the end of the protobuf so the fields are sorted in ascending order of field numbers